docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: docs
+title: Browser Support
+permalink: /documentation/using-testcafe/common-concepts/browser-support.html
+---
+# Browser Support
+
+TestCafe allows you to test against different web browsers.
+The most popular browsers are [supported directly](#directly-supported-browsers),
+while others require a [browser provider plugin](#browser-provider-plugins).
+
+The topic contains the following sections.
+
+* [Directly Supported Browsers](#directly-supported-browsers)
+* [Browser Provider Plugins](#browser-provider-plugins)
+    * [Published Browser Provider Plugins](#published-browser-provider-plugins)
+* [Selecting Browsers for Testing](#selecting-browsers-for-testing)
+
+## Directly Supported Browsers
+
+TestCafe supports testing against the following browsers out-of-the-box.
+
+Browser           |Version | Short Browser Name
+----------------- | ------ | -------------------
+Chromium          | 22+    | `chromium`
+Google Chrome     | 22+    | `chrome`
+Internet Explorer | 9+     | `ie`
+Microsoft Edge    |        | `edge`
+Mozilla Firefox   | 17+    | `firefox`
+Opera             | 11+    | `opera`
+Safari            | 6+     | `safari`
+
+## Browser Provider Plugins
+
+If you need to use a browser that is not [supported directly](#directly-supported-browsers), create a [browser provider plugin](../../extending-testcafe/custom-browser-provider-plugin/).
+
+### Published Browser Provider Plugins
+
+After a browser provider plugin is created, it can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; for example, `testcafe-browser-provider-saucelabs`.
+
+You can search for available browser provider packages on npm: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
+
+For example, there are provider packages for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+
+* [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+* [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs)
+
+You can install browser providers packages from npm as you would install any other plugin. See [Installing Plugins](../../extending-testcafe/index.md#installing-plugins).
+
+## Selecting Browsers for Testing
```

Comment:
Sounds way too general. Like *Selecting Personnel for Training*.

Maybe something with words like `specify` and `test task`?